src/main/java/org/opentripplanner/routing/graph_finder/DirectGraphFinder.java

```diff
@@ -0,0 +1,65 @@
+package org.opentripplanner.routing.graph_finder;
+
+import com.beust.jcommander.internal.Lists;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.opentripplanner.common.geometry.GeometryUtils;
+import org.opentripplanner.common.geometry.SphericalDistanceLibrary;
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.model.TransitMode;
+import org.opentripplanner.routing.RoutingService;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.impl.StreetVertexIndex;
+import org.opentripplanner.routing.vertextype.TransitStopVertex;
+
+import java.util.List;
+
+/**
+ * A Graph finder used in conjunction with a graph, which does not have a street network included.
+ * Also usable if performance is more important or if the "as the crow flies" distance id required.
+ */
+public class DirectGraphFinder implements GraphFinder {
+
+  private static GeometryFactory geometryFactory = GeometryUtils.getGeometryFactory();
+
+  private StreetVertexIndex streetIndex;
+
+  public DirectGraphFinder(Graph graph) {
+    this.streetIndex = graph.streetIndex != null ? graph.streetIndex : new StreetVertexIndex(graph);
+  }
+
+  /**
+   * Return all stops within a certain radius of the given vertex, using straight-line distance independent of streets.
+   * If the origin vertex is a StopVertex, the result will include it.
+   */
+  @Override
+  public List<StopAtDistance> findClosestStops(double lat, double lon, double radiusMeters) {
+    List<StopAtDistance> stopsFound = Lists.newArrayList();
+    Coordinate c0 = new Coordinate(lon, lat);
+    for (TransitStopVertex ts1 : streetIndex.getNearbyTransitStops(c0, radiusMeters)) {
```

Comment:
For the inner most loop iterator variable, the standard is to use `it`, which is also used in lambdas.

Reply:
This was just lifted off from here https://github.com/opentripplanner/OpenTripPlanner/pull/3122/files#diff-7aa5baeb59c0d0b53802ce6ec0efe627L216-L233. I'll update it.